WeWatch/WeWatch/Service/Webservice.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Webservice.swift
+//  WeWatch
+//
+//  Created by Anton on 30/11/2024.
+//
+
+import Foundation
+
+enum AuthenticationError: Error {
+    case invalidCredentials
+    case custom (errorMessage: String)
+}
+
+struct LoginRequestBody: Codable {
```

Comment:
Remember that if this is the body of the request, you will only need an `Encodable` conformance. Otherwise you will need `Decodable`.